docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -302,7 +302,7 @@ As documented in [Suggested changes for providers](#suggested-changes-for-provid
   ```go
   import (
       ...
-      clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+      clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta1"
```

Comment:
Should we document the API movals in this doc? (OK as follow-up)

Reply:
I was planning to push another commit with that in a bit